Remotion/ObjectBinding/Web/Res/Themes/NovaViso/HTML/BocEnumValue.css

```
@@ -3,76 +3,92 @@
 span.bocEnumValue,
 div.bocEnumValue.readOnly
 {
-  width: 15em;
-  position: relative;
-  display: inline-block;
-  vertical-align: middle;
-  overflow: hidden;
+  display: inline-flex;
+  width: var(--remotion-themed-control-width);
+  min-height: var(--remotion-themed-min-height);
 }
 
-.bocEnumValue.readOnly
+.bocEnumValue td
 {
-  /* Ensures that the outline is visible in Firefox. */
-  overflow: visible;
+  white-space: nowrap;
 }
 
-.bocEnumValue.disabled
+.bocEnumValue td label
 {
+  padding: var(--remotion-themed-padding);
 }
 
-span.bocEnumValue select
+div.bocEnumValue td,
+div.bocEnumValue td span
 {
-  width: 100%;
-  height: 1.68em;
+  display: inline-flex;
+  align-items: center;
+  box-sizing: border-box;
 }
 
-span.bocEnumValue select[size]
+div.bocEnumValue input
 {
-  height: auto;
+  /* Has to be set explicitely for input. */
+  margin: 0;
 }
 
-span.bocEnumValue select[size="1"]
+div.bocEnumValue input:focus
 {
-  height: 1.68em;
+  /* In order to have an outline for the radio buttons as well. */
+  outline: auto;
 }
 
-.bocEnumValue td
+div.bocEnumValue input:hover
 {
-  white-space: nowrap;
+  cursor: pointer;
 }
 
-.bocEnumValue td label
+div.bocEnumValue input:disabled
+{
+  cursor: default;
+}
+
+span.bocEnumValue select:hover,
+span.bocEnumValue option:hover
+{
+  background-color: var(--color-highlight);
+}
+
+span.bocEnumValue select
 {
-  vertical-align: text-bottom;
+  width: 100%;
 }
 
-.bocEnumValue.readOnly span[tabindex]
+span.bocEnumValue option
 {
-  /* Ensures that diacritic marks can be rendered without being cut off. 
-     Using line-height instead of padding works better with BocEnumValue. When using padding, the text is not aligned with the label. */
-  /*line-height: 1.5em;*/
-  /* Ensures that the outline is rendered outside of the space reserved for diacritic marks. */
-  display: inline-block;
+  padding: calc(var(--remotion-themed-padding) * 0.75);
+  border-radius: var(--remotion-themed-border-radius);
 }
 
-.bocEnumValue.readOnly span[tabindex]:first-child
+span.bocEnumValue.disabled option
 {
-  /* Ensures that diacritic marks can be rendered without being cut off. */
-  padding-top: 0.2em;
+  background-color: var(--color-disabled-background);
+  color: var(--color-disabled-text);
+  text-decoration: none;
 }
 
-.bocEnumValue.readOnly span[tabindex]:last-child
+.bocEnumValue.readOnly span
 {
-  /* Ensures that diacritic marks can be rendered without being cut off. */
-  padding-bottom: 0.2em;
+  box-sizing: border-box;
+  padding: var(--remotion-themed-padding) 0;
```

I think it's better to not use the shorthand property if the property has more than one value.
```suggestion
  padding-top: var(--remotion-themed-padding);
  padding-bottom: var(--remotion-themed-padding);
  padding-left: 0;
  padding-right: 0;
```